pkgs/development/tools/build-managers/ninja/default.nix

```diff
@@ -62,6 +48,10 @@ stdenv.mkDerivation rec {
     CFLAGS="$CFLAGS_FOR_BUILD" \
     CXXFLAGS="$CXXFLAGS_FOR_BUILD" \
     LDFLAGS="$LDFLAGS_FOR_BUILD" \
+
+    # write rebuild args to file after bootstrap
+    substituteInPlace configure.py --replace "subprocess.check_call(rebuild_args)" "open('rebuild_args','w').write(rebuild_args[0])"
+
```

Comment:
Why this change.

Reply:
The *patch* is not applied anymore, so it makes sense to move the snippet from postPatch to buildPhase. 

Additionally ninja can not build at all without this substituteInPlace command, even with or without the patch applied.

Reply:
`postPatch` has nothing to do with patches, you can leave it there.

Reply:
I've updated it, please check.

Reply:
> I've updated it, please check.

The version update is accidentally dropped?

Reply:
Thanks for noticing.